vmdb/app/models/miq_event.rb

```diff
@@ -170,173 +147,18 @@ def self.raise_event_for_children(target, raw_event, inputs={})
       next unless target.respond_to?(assoc)
       children = target.send(assoc)
       children.each do |child|
-        $log.info("MIQ(Event.raise_event_for_children): Raising Event [#{child_event}] for Child [(#{child.class}) #{child.name}] of Parent [(#{target.class}) #{target.name}]")
+        $log.info("MIQ(EventDefinition.raise_event_for_children): Raising Event [#{child_event}] for Child [(#{child.class}) #{child.name}] of Parent [(#{target.class}) #{target.name}]")
```

Comment:
I think log message was wrong to begin with and should say "MIQ(MiqEventDefinition..."